hrm-domain/hrm-core/profile/profileDataAccess.ts

```diff
@@ -311,7 +311,7 @@ export const associateProfileToProfileFlag =
             profileId,
           });
 
-          const profile = await getProfileById(t)(accountSid, profileId);
+          const profile = await getProfileById(t)(accountSid, profileId, true);
```

Comment:
I think is easier to understand named parameters, e.g.
```suggestion
          const profile = await getProfileById(t)({ accountSid, profileId, includeFullRelations: true });
```

I can keep going to the function definitions, but I will also continue leaving this comments :laughing: